lib/finders/people.json

```diff
@@ -0,0 +1,30 @@
+{
+  "base_path": "/government/people",
+  "title": "All ministers and senior officials on GOV.UK",
+  "description": "",
+  "locale": "en",
+  "document_type": "finder",
+  "schema_name": "finder",
+  "publishing_app": "whitehall",
+  "rendering_app": "finder-frontend",
+  "details": {
+    "document_noun": "person",
+    "default_order": "title",
+    "facets": [],
+    "filter": {
+      "format": "person"
```

Comment:
I didn't know it was being deprecated. From what I can see in https://www.gov.uk/api/search.json?filter_format=person, the only field I can use in order for `finder-frontend` to perform filtering is `format`...